### PR TITLE
PLANET-6525 Implement Take Action pattern layout

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6525](https://jira.greenpeace.org/browse/PLANET-6525)

### Testing

You can add the Take Action pattern layout to a page either via the block inserter or the page creation modal, then you can edit it as you wish!

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [pandora](https://www-dev.greenpeace.org/test-pandora) | [admin](https://www-dev.greenpeace.org/test-pandora/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-pandora/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-pandora) | [composer-local.json](https://github.com/greenpeace/planet4-test-pandora/blob/main/composer-local.json)

:watch: 2022.08.15 08:09:05

File: classes/patterns/class-takeaction.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$classname = self::get_classname();
+
+		return [
+			'title'      => __( 'Take Action', 'planet4-blocks-backend' ),
```
Comment (sagarsdeshmukh) [inline classes/patterns/class-takeaction.php:36]: As like the block names we can disallow block pattern names translation. :) (if admin translate block pattern name then new editor/admin user may confuse the block pattern name on site and in handbook documentation)

**Review by sagarsdeshmukh — APPROVED** [review]
Tiny suggestion about translation of block pattern name, other than that PR looks good to me 👍

File: classes/patterns/class-takeaction.php (hunk 1 above)
Comment (mleray) [reply classes/patterns/class-takeaction.php]: That's a good point! I've added a [commit](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/925/commits/a47ddeb304305ed567ffbc5ec0b577413830aaa2) removing translations not only from this pattern, but from all of the existing ones

**Review by sagarsdeshmukh — APPROVED** [review]
LGTM 👍

**Pull request merged**